lib/runner.js

```diff
@@ -261,6 +234,156 @@ module.exports = class Runner {
 		return title + header + failureBody;
 	}
 
+	/**
+	 * Generates a body to be used in an email report that only contains failures and slows
+	 *
+	 * @param {String} suiteName Name of the test suite being ran
+	 * @param {Object} data Data that was generated from the Mocha test run
+	 *
+	 * @returns {String} Body to be used in the report body
+	 */
+	generateMinimalReport (suiteName, data) {
```

Comment:
There is a lot of duplicate code here with the existing report generation, seems to me like the minimal report could just be an optional third param in the existing generateReport function